src/harness/BenchmarkDotNet.Extensions/ExclusionFilter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Running;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    class ExclusionFilter : IFilter
+    {
+        private string filter;
```

Comment:
nit: it can be readonly
```suggestion
        private readonly string filter;
```